### PR TITLE
Adding french Android Meta

### PR DESCRIPTION
**Pull request by BoFFire**
Adding french Android meta full description.

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
-    <string name="settings_change_address">Changement d'adresse</string>
+    <string name="settings_change_address">Changement d’adresse</string>
     <string name="settings_change_password">Changer le mot de passe</string>
     <string name="settings_nightmode">Mode nuit</string>
     <string name="settings_ignore_unsaved">Ignorer les appelants inconnus</string>
-    <string name="about_source">Code source:</string>
-    <string name="about_author">Développeurs:</string>
-    <string name="about_license">Licence:</string>
+    <string name="about_source">Code source :</string>
+    <string name="about_author">Développeurs :</string>
```
Comment (mwarning) [inline app/src/main/res/values-fr/strings.xml]: I was about to ask you if that space is a french thing. But a friend next to be pulled out a french book and assured me. XD

**Comment by mwarning** [comment]
Thanks!

**Comment by mwarning** [comment]
@BoFFire Some texts were added in 3.0.0. Maybe you can provide a translation for those as well: https://github.com/meshenger-app/meshenger-android/blob/master/app/src/main/res/values-fr/strings.xml
- thanks

**Comment by BoFFire** [comment]
Hi @mwarning I'll do it right now. Thanks :)

**Comment by mwarning** [comment]
awesome

**Pull request merged**